### ISSUE TITLE
mc service status should follow new json format.

### ISSUE THREAD

**Issue by harshavardhana** [minio/mc#1963]
## Expected behaviour

```
{
    "status": "success",
    "service": "online|offline",
    "server": {
        "version": "RELEASE.<xxx>",
        "commitID": "<commit-id"
    },
    "storage": {
        "total": 675875831808,
        "free": 595090522112,
        "backend": {
            "format": "XL",
            "onlineDisks": 8,
            "offlineDisks": 0
        }
    }
}
```

## Actual behaviour
```
 mc admin service --json status play | jq .
{
  "status": "success",
  "storageInfo": {
    "total": 675875831808,
    "free": 595089997824,
    "backend": {
      "backendType": "XL",
      "onlineDisks": 8,
      "offlineDisks": 0
    }
  }
}
```

## Steps to reproduce the behaviour

```
mc admin service --json status play/
```

## mc version
Current master

**Comment by vadmeste** [comment]
> "service": "online|offline",

What does service online/offline mean here? like server initialized?

**Comment by harshavardhana** [comment]
> What does service online/offline mean here? like server initialized?

It means that "online" meaning the server is responding to API requests, "offline" meaning server might be online but it is not responding to API requests.

**Comment by vadmeste** [comment]
Fixed by #1979
